app/src/main/java/com/capstone/weatherapp/DetailsViewModel.kt

```diff
@@ -1,26 +1,45 @@
 package com.capstone.weatherapp
 
+import androidx.lifecycle.LiveData
 import androidx.lifecycle.ViewModel
 
 import androidx.lifecycle.MutableLiveData
+import androidx.lifecycle.ViewModelProvider
 import androidx.lifecycle.viewModelScope
 import kotlinx.coroutines.launch
 
 private const val APIKEY = "6849ae760f417fb8188f4bb7fd0d92fc"
 private const val UNITS = "imperial"
 
-class DetailsViewModel : ViewModel() {
-    private val _city = MutableLiveData<SingleCityResponse?>()
-    val city: MutableLiveData<SingleCityResponse?>
+class DetailsViewModel(private val weatherRepo: WeatherRepository) : ViewModel() {
+    private val _city = weatherRepo.singleCityData
+    val city: MutableLiveData<SingleCityResponse>
```

Comment:
Same thing as the above comment from the other viewModel, don't need the private one anymore